spec/system/candidate_interface/candidate_entering_work_history_spec.rb

```diff
@@ -4,55 +4,65 @@
   include CandidateHelper
 
   scenario 'Candidate submits their work history' do
-    given_i_am_signed_in
-    and_i_visit_the_site
+    FeatureFlag.activate('work_breaks')
 
-    when_i_click_on_work_history
-    then_i_should_see_a_list_of_work_lengths
+    Timecop.travel(Time.zone.local(2019, 12, 1)) do
```

Comment:
Can we avoid wrapping this in a block? You could use `Timecop.travel` without the block, use an `around` statement, or only wrap the code in a step.

Reply:
I realized that, in that case, we do not needed to freeze the time.